options/linux/generic/sys-prctl-stubs.cpp

```diff
@@ -1,12 +1,32 @@
 
+#include <stdarg.h>
+#include <errno.h>
 #include <bits/ensure.h>
 #include <sys/prctl.h>
 
 #include <mlibc/debug.hpp>
 
-int prctl(int, ...) {
-	mlibc::infoLogger() << "\e[31mmlibc: prctl() is not implemented correctly\e[39m"
-			<< frg::endlog;
+#include "mlibc/linux-sysdeps.hpp"
+
+int prctl(int op, ...) {
```

Comment:
Why do we use varargs and not simply take 4 longs?

Reply:
`musl` does also take varargs, while the man page talks about `unsigned long`s. I'm unsure what to follow here, but varargs feel like the less troublesome variant.